package.json

```diff
@@ -528,11 +528,10 @@
     "@types/glob": "^7.1.2",
     "@types/gulp": "^4.0.6",
     "@types/gulp-zip": "^4.0.1",
-    "@types/hapi__cookie": "^10.1.1",
-    "@types/hapi__h2o2": "^8.3.2",
-    "@types/hapi__hapi": "^20.0.2",
-    "@types/hapi__inert": "^5.2.2",
-    "@types/hapi__podium": "^3.4.1",
```

Comment:
types are included in the hapi/podium package now.  I checked the others, this was the only one.